functions.sh

```diff
@@ -359,3 +359,51 @@ function tests_updated() {
   fi
   return 0
 }
+
+function arch_to_buildx_platform() {
+  local arch
+  local platform
+  arch=$1
+  shift
+
+  case $arch in
+    amd64)
+      platform="amd64"
+      ;;
+    arm32v6)
+      platform="arm/v6"
+      ;;
+    arm32v7)
+      platform="arm/v7"
```

Comment:
Not sure this is right, it you're looking to match with the value in https://nodejs.org/dist/v10.22.1/

Reply:
I'm trying to convert the arches in the `architectures` file into an arch that [docker buildx supports](https://github.com/docker/cli/blob/1f6a1a438c4ae426e446f17848114e58072af2bb/cli/command/manifest/util.go#L22).